cognite_toolkit/_cdf_tk/prototypes/interactive_init.py

```diff
@@ -1,231 +1,44 @@
-        ("highlighted", "fg:#673ab7 bold"),  # pointed-at choice in select and checkbox prompts
-        ("selected", "fg:#673ab7"),  # style for a selected item of a checkbox
-        ("separator", "fg:#cc5454"),  # separator in lists
-        ("instruction", ""),  # user instructions for select, rawselect, checkbox
-        ("text", ""),  # plain text
-        ("disabled", "fg:#858585 italic"),  # disabled choices for select and checkbox prompts
-    ]
-)
-
-
-def get_packages() -> dict[str, dict[str, Any]]:
-    return {
-        "quickstart": {
-            "title": "Quick Start: A set of modules for a CDF quick start project.",
-            "items": {
-                "sap_data_pipeline": {
-                    "title": "SAP Data Pipeline",
-                },
-                "pi_data_pipeline": {
-                    "title": "PI Data Pipeline",
-                },
-                "mqtt_data_pipeline": {
-                    "title": "MQTT Data Pipeline",
-                },
-                "files_contextualization": {
-                    "title": "Files Contextualization",
-                },
-                "asset_data_transformation": {
-                    "title": "Asset Data Transformation",
-                },
-                "infield": {
-                    "title": "Infield",
-                },
-            },
-        },
-        "examples": {
-            "title": "Examples: a set of example modules for inspiration",
-            "items": {
-                "cdf_data_pipeline_asset_valhall": {"items": {}},
-                "cdf_data_pipeline_files_valhall": {"items": {}},
-            },
-        },
-        "reference": {
-            "title": "All supported resources as reference",
-            "items": {"workflow": {}, "transformations": {}, "functions": {}, "groups": {}},
-        },
-    }
+from cognite_toolkit._cdf_tk.prototypes.init import InitCommand
 
 
 class InteractiveInit(typer.Typer):
     def __init__(self, *args, **kwargs) -> None:  # type: ignore
         super().__init__(*args, **kwargs)
         self.command()(self.interactive)
 
-    def build_tree(self, item: Union[dict, list], tree: Tree) -> None:
-        if isinstance(item, dict):
-            for key, value in item.items():
-                subtree = tree.add(key)
-                for subvalue in value:
-                    if isinstance(subvalue, dict):
-                        self.build_tree(subvalue, subtree)
-                    else:
-                        subtree.add(subvalue)
-
-    def create(self, init_dir: str, selected: dict[str, list[str]], mode: str | None) -> None:
-        pass
-
     def interactive(
         self,
         ctx: typer.Context,
-        init_dir: Annotated[
+        arg_init_dir: Annotated[
             Optional[str],
             typer.Option(
+                "--init-dir",
                 help="Directory path to project to initialize or upgrade with templates.",
```

Comment:
Expected this to be an positional argument.
```bash
cdf-tk init my_project
```

Instead of 
```bash
cdf-tk init --init-dir my_project
```

Reply:
It is not really supposed to be an argument, but I decided to support it for convenience. Unsure if it can be both optional and positional...